lets-encrypt_acme_renew_script/server_setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+document_root=/var/www/html # change to your document root
+nginx_location=$(which nginx)
+nginx_config_location=/etc/nginx #change if this is not your config directory
+scripts_location=/usr/local/scripts # change if necessary
+certs_location=ssl
+mkdir -p ${scripts_location}
+useradd -r -d /nodir -s /usr/sbin/nologin acme
+cd /usr/local/scripts/
+git clone https://github.com/frezbo/acme-tiny.git
```

Comment:
Hi, Aplanas,
I have mentioned in the README to read and understand the script before deploying it. Thanks.